MyHorizons/Data/Save/MainSaveFile.cs

```diff
@@ -3,35 +3,33 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using MyHorizons.Data.TurnipsData;
 
 namespace MyHorizons.Data.Save
 {
     public sealed class MainSaveFile : SaveBase
     {
-        private static MainSaveFile? _saveFile;
         private readonly List<PlayerSave> _playerSaves = new List<PlayerSave>();
         
-        public static MainSaveFile Singleton() => _saveFile ?? throw new NullReferenceException("Main save file was null!");
-
         public int NumPlayers => _playerSaves.Count;
         public readonly Town? Town;
+        public readonly Turnips? Turnips;
```

Comment:
Ideally this should be in the Town class as the idea is for it to encompass everything town related and turnip prices are per-town.